docker/README.md

```diff
@@ -46,11 +46,41 @@ You may need to temporarily set your `siteurl` and `home` `wp_option`s to the ne
 
 Visit the `<url>` , login and setup WCPay.
 
+### Setting up an additional Docker environment
+
+If you need to set up a different local environment alongside the default one, here are the steps to follow:
+1. Clone the repository to a new directory and navigate to it.
+2. Run `npm install && composer install` to install the dependencies.
+3. Create a `docker-compose.override.yml` file in the new directory with the following contents:
+    ```
+    services:
+      wordpress:
+        container_name: woopayments_apiv2_wordpress # Change the container name.
+        build:
+          args:
+            - XDEBUG_REMOTE_PORT=9003 # Change the xDebug port.
+        ports: !override # This will override the default ports rather than appending to them.
+          - "8084:80" # Change the HTTP port.
+        volumes:
+          - /Users/vladolaru/Work/a8c/woocommerce-develop/plugins/woocommerce:/var/www/html/wp-content/plugins/woocommerce
+          - /Users/vladolaru/Work/a8c/woocommerce-develop/plugins/woocommerce-beta-tester:/var/www/html/wp-content/plugins/woocommerce-beta-tester
```

Comment:
@vladolaru These paths are specific to your machine. Can we use generic ones? Maybe switch to relative paths?

Reply:
Good point, Achyuth! Let me give it a try with relative paths.

Reply:
On second thought, those are things from a different story (setting up WooPayments local dev to use a cloned WooCommerce repo). It is best to [remove them](https://github.com/Automattic/woocommerce-payments/pull/9256/commits/187dda3f1012615d8ac16504890a372487667e2f).

Reply:
Nice :)